mailpoet/readme.txt

```diff
@@ -152,7 +152,9 @@ To improve user experience, MailPoet may use the following 3rd party libraries i
 
 * Google Fonts - used in Form Editor which you can use to customize your forms, and in the Email Editor to style emails. This can be individually [disabled by a filter](https://kb.mailpoet.com/article/332-how-to-disable-google-fonts)
 
-* HelpScout - used to show help tooltips throughout the plugin, easily access documentation, and contact our customer support team. This functionality may load scripts from [https://beacon-v2.helpscout.net/](https://beacon-v2.helpscout.net/)
+* Zendesk - used to show help tooltips throughout the plugin, easily access documentation, and contact our customer support team. This functionality may load scripts from [https://static.zdassets.com/ekr/snippet.js](https://static.zdassets.com/ekr/snippet.js)
```

Comment:
`used to show help tooltips throughout the plugin, easily access documentation, and contact our customer support team.` Is it still true? When I tried the tooltips (green badges) on the create email page, it now links to KB articles and it used to open the HelpScout widget with the post. 
Also, if I'm not mistaken, it doesn't contain links to documentation.
So maybe we should just say `used to show widget for contacting our customer support team.` ?